### DISCUSSION TITLE
<freeswitch新增模块amqp>

### DISCUSSION THREAD

**Discussion by Gebriel18** [community/community#76809] [Discussions]
### Select Topic Area

Question

### Body

在安装完成freeswitch后，更改modules.conf 去掉mod_amqp注释，然后./configure ,并且执行make mod_amqp-install 报错如下
![QQ图片20231124103244](https://github.com/community/community/assets/99950407/73ebed83-612f-4cde-9a2f-11164cecd42c)
然后 我使用
./configure \
    --enable-pic \
    --enable-static \
    --enable-shared \
    --as=yasm \
    --target=generic-gnu
报错没有as选项

**Comment by Akash1134** [comment]
Hi @Gebriel18 , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
